doc/src/message_access.md

```diff
@@ -0,0 +1,127 @@
+# Contract Messages with Special Access Needs
+Only those contracts that has at least one execute message with restricted access are included in this part.
```

Comment:
```suggestion
Only those contracts that have at least one execute message with restricted access are included in this part.
```
Before this section, add an explanation what the possible permission levels are (a message can either be called by anyone, only by governance, only by the contract admin or by either governance or admin)